.github/workflows/release_draft.yml

```
@@ -119,6 +119,7 @@ jobs:
               {"title": "Bugfixes 🛠", "label": "bug"}
             ]
           warnings: true
+          skip-release-notes: no RN
```

```suggestion
          skip-release-notes: "no RN"
```

I think It should be in quotes